.github/workflows/ci.yml

```diff
@@ -89,4 +89,4 @@ jobs:
       run: ninja
     - name: Run AVIF Tests
       working-directory: ./build
-      run: ctest -j 2
+      run: ctest -j $(getconf _NPROCESSORS_ONLN)
```

Comment:
My experiments showed `getconf _NPROCESSORS_ONLN` returns 2 and 3 in the virtual machines running the ubuntu-latest and macos-latest os images, respectively.

`nproc` and `getconf _NPROCESSORS_ONLN` return the same values on my Linux workstation (72) and the virtual machine running ubuntu-latest (2).

On my Mac Pro desktop, `getconf _NPROCESSORS_ONLN` and `sysctl -n hw.logicalcpu` both return 12, whereas `sysctl -n hw.physicalcpu`  returns 6.